eox_nelp/pearson_vue/pipeline.py

```diff
@@ -613,3 +614,43 @@ def get_enrollment_from_id(enrollment_id, enrollment=None, **kwargs):  # pylint:
             pass
 
     return {}
+
+
+def save_audit_pipeline(pipeline_backend, last_step, cdd_import=None, ead_import=None, **kwargs):
```

Comment:
As you mentioned this is not a pipeline step, so I don't understand why you included this in the **pipeline** file, anyway I wondering why you split the logic into this method and audit pipeline decorator